### DISCUSSION TITLE
Cannot synced from 2 hours.

### DISCUSSION THREAD

**Discussion by leonneonn** [Chia-Network/chia-blockchain#1637] [General]
Do you have problems too or this problem only I have? All were ok until now. I'm making new plot may be that's why?
Status
Syncing 63612/65054
Connection Status
Connected
Network Name
mainnet
Peak Height
63612
Peak Time
04/01/2021 11:10:17
 I cant move from that point from many hours.

**Comment by dorkmo** [comment]
might have overloaded your computer with the plotting? Its recomended to farm on one pc and plot on a different one

**Comment by leonneonn** [comment]
you mean I should later move plots to another one computer? How I can move them and connect witch chia app? Can I have same account for ploting and same account in another computer to farm?  
I also stop ploting now and deleted plots and nothing change. Should I uninstal and instal chia again?

**Reply by dorkmo** [reply]
you can use the same 24 words key to login on a different pc for farming, then itll be just the same reward key.
you can just copy and paste the files from one drive to the other. just make sure to add the plot directory.
i usually restart my computer if i stop plotting or need to change anything.

**Reply by leonneonn** [reply]
"just make sure to add the plot directory" Just by adding plot and then chose done block already?   So what you think I should now uninstall and install again chia?  One more question  is that posible to restume plot  what was not finished and continiuse from that point or every time have to start from begining?

**Reply by hoffmang9** [reply]
Plots are not yet resume-able.

**Comment by leonneonn** [comment]
Just uninstaled chia block chain. I started setup 1.0.3 and I was thinking this will instal chia again but not. Just started connect to wallet and same story wallet not sync and full node status the same Syncing 63612/65793            " 63612 " this is not moving, not syncing from almoust 12 hours. Any ideas how to fix this? how I can uninstall all and instal again and sync from begining?


Last update
Now start works. What I did?  Some how I dont know how  but time at my computer changed 2 hours. This didnt hapend for me from more then few years. So I think that chia couldnt sync with me because time at windows was not correct. I just found it now and make time like should be. Is another option to, I install chia on my laptop and import acc so maybe this some how make to sync my accound.  
May be this will help someone of you in future to save a lot of time. Thank to all who came and try to help me.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
